src/main/java/com/julienviet/pgclient/impl/PreparedBatchCommand.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2017 Julien Viet
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.julienviet.pgclient.impl;
+
+import com.julienviet.pgclient.codec.Message;
+import com.julienviet.pgclient.codec.decoder.message.BindComplete;
+import com.julienviet.pgclient.codec.decoder.message.NoData;
+import com.julienviet.pgclient.codec.decoder.message.ParameterDescription;
+import com.julienviet.pgclient.codec.decoder.message.ParseComplete;
+import com.julienviet.pgclient.codec.decoder.message.ReadyForQuery;
+import com.julienviet.pgclient.codec.encoder.message.Bind;
+import com.julienviet.pgclient.codec.encoder.message.Describe;
+import com.julienviet.pgclient.codec.encoder.message.Execute;
+import com.julienviet.pgclient.codec.encoder.message.Parse;
+import com.julienviet.pgclient.codec.encoder.message.Sync;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.ext.sql.UpdateResult;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.julienviet.pgclient.codec.util.Util.*;
+
+/**
+ * @author <a href="mailto:emad.albloushi@gmail.com">Emad Alblueshi</a>
+ */
+
+class PreparedBatchCommand extends UpdateCommandBase {
```

Comment:
please keep PreparedUpdateCommand and consider that an update is a batch of a single unit.

Reply:
Yes now we have only PreparedUpdateCommand and can be used for batch and update